flex/engines/graph_db/database/manager.cc

```diff
@@ -0,0 +1,487 @@
+/** Copyright 2020 Alibaba Group Holding Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "flex/engines/graph_db/database/manager.h"
+#include <nlohmann/json.hpp>
+#include <string>
+#include <unordered_map>
```

Comment:
The included headers maybe organized as follows
0. manager.h
1. headers from std 
2. headers from third party
3. headers from flex

Reply:
I've organised it in this way